tests/integration/test_chat.py

```diff
@@ -890,3 +890,14 @@ def test_complete_event_emitted_with_step_without_responses(self, mock_publish):
         self.click_button('OK')
         self.wait_for_ajax()
         mock_publish.assert_called_with(ANY, 'xblock.chat.complete', {'final_step': '4'})
+
+    @patch('chat.chat.ChatXBlock.chat_complete')
+    def test_ping_handler_when_chat_is_complete(self, mock_handler):
+        mock_handler.return_value = {}
+        self.configure_block(yaml_final_steps)
+        self.element = self.go_to_view('student_view')
+        self.click_button('Response that points to non-existing step')
```

Comment:
It would be good to test [this case](https://github.com/replaceafill/xblock-chat/blob/318c3e410d96a3fd7fdfc56b86514606d8bb15d6/tests/integration/test_chat.py#L226), too.